### PR TITLE
fix 5 api_cn documents

### PR DESCRIPTION
**Pull request by huangjun12**
test=document_preview
1. hard_swish
2. huber_loss
3. linear_lr_warm_up
4. lrn
5. roi_pool

### CODE REVIEW

**Comment by huangjun12** [comment]
hard_swish

**Comment by huangjun12** [comment]
huber_loss
linear_lr_warmup
lrn
roi_pool

File: doc/fluid/api_cn/layers_cn/hard_swish_cn.rst [hunk 1]
```diff
@@ -5,31 +5,45 @@ hard_swish
...
+    DATATYPE='float32'
+    shape = [1,4]
+
+    x_data = np.array([i for i in range(1,5)]).reshape(shape).astype(DATATYPE)
+
+    x = fluid.layers.data(name="x", shape=shape, dtype=DATATYPE)
     y = fluid.layers.hard_swish(x)
 
+    place = fluid.CPUPlace()
+    place = fluid.CUDAPlace(0)
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/hard_swish_cn.rst:40]: CPUPlace删了吧

Comment (huangjun12) [reply doc/fluid/api_cn/layers_cn/hard_swish_cn.rst:40]: Done

**Review by wanghaoshuang — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Pull request merged**